bin/schema-reverse

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env qore
+%new-style
+%exec-class Main
+%requires SchemaReverse
+%requires SqlUtil
+
+const VERSION = "0.1";
+
+const OBJECT_OPTIONS = (
+        "sequences"     : "seq,q:s",
+        "tables"        : "table,t:s",
+        "types"         : "type,y:s",
+        "views"         : "view,v:s",
+        "mviews"        : "mview,m:s",
+        "procedures"    : "proc,p:s",
+        "functions"     : "func,f:s",
+        "packages"      : "pkg,g:s",
+    );
+
+const OPTIONS = (
```

Comment:
please modify according to https://github.com/qorelanguage/qore/wiki/ProTips#command-line-argument-processing